cmd/root.go

```diff
@@ -4,11 +4,12 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/Docplanner/github-flow-manager/flow-manager"
-	"github.com/olekukonko/tablewriter"
-	"github.com/spf13/cobra"
 	"strings"
 	"time"
+
+	flow_manager "github.com/Docplanner/github-flow-manager/flow-manager"
```

Comment:
Sorry, I'm not familiar with go syntax but is this change ok?

Reply:
Thx I did not notice it... in local VS code with GO applies automatic changes on file save